src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,16 @@
-export const TodoInfo = () => {};
+import React from 'react';
+import { Users, Todo } from '../TodoList';
+import { UserInfo } from '../UserInfo/UserInfo';
+
+type Props = {
+  usersFromServer: Users[],
+  todo: Todo,
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo, usersFromServer }) => {
+  return (
+    <article data-id={todo.id} className="TodoInfo TodoInfo--completed">
```

Comment:
you must check this todo completed or not, don't use TodoInfo--completed always
use classnames library

Reply:
agree